.github/meta.yaml

```diff
@@ -37,7 +37,7 @@ outputs:
         - requirements-parser >=0.2.0
         - shap >=0.40.0
         - texttable >=1.6.2
-        - woodwork >=0.16.2, < 0.17.0
+        - woodwork >=0.18.0
```

Comment:
Supporting minimum of `woodwork==0.18.0`; I wanted to make sure the `replace_nan` optimizations were supported at a minimum but feel free to push back on this